src/components/10-atoms/button/pw.ui.test.js

```diff
@@ -0,0 +1,36 @@
+const { chromium } = require('playwright-chromium');
+
+const host = process.env.TEST_HOST_STORYBOOK_URL;
+
+let browser;
+let page;
+
+const config = {
+  // devtools: true
+};
+
+beforeAll(async () => {
+  browser = await chromium.launch(config);
```

Comment:
Is it possible to launch browser only once? So that all the test-files do not need to start chromium on their own. A test should be "dump" and should just say "browser start" instead of "chromium start".

Reply:
I could add a class that abstracts this method, but imho this introduces more complexity that is being solved.

Reply:
Maybe this makes sense, when we start having shared test-configs and so on. I will give it a try.